modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -213,9 +218,28 @@ private String _addNewReference(String content, String newReference) {
 		return content;
 	}
 
+	private String _addOrReplaceGenerics(
+		List<String> genericNames, List<String> newGenericNames,
+		String newMethodCall) {
+
+		return _addOrReplacePattern(
+			StringPool.GREATER_THAN, newMethodCall, newGenericNames,
+			genericNames, "generic#");
+	}
+
 	private String _addOrReplaceParameters(
-		String newMethodCall, List<String> parameterNames,
-		List<String> newParameterNames) {
+		String newMethodCall, List<String> newParameterNames,
+		List<String> parameterNames) {
+
+		return _addOrReplacePattern(
+			StringPool.CLOSE_PARENTHESIS, newMethodCall, newParameterNames,
+			parameterNames, "param#");
+	}
+
+	private String _addOrReplacePattern(
+		String lastCharacter, String newMethodCall,
+		List<String> newParameterNames, List<String> parameterNames,
+		String prefix) {
```

Comment:
@TamyrisTorres, since we're already using the term `pattern` to reference the regex, let's try different names like below:
```
private String _addOrReplaceTypeParameters(
	String newMethodCall, List<String> newTypeParameterNames,
	List<String> typeParameterNames) {

	return _addOrReplaceParameters(
		StringPool.GREATER_THAN, newMethodCall, newTypeParameterNames,
		typeParameterNames, "typeParam#");
}

private String _addOrReplaceMethodParameters(
	List<String> methodParameterNames, String newMethodCall,
	List<String> newMethodParameterNames) {

	return _addOrReplaceParameters(
		StringPool.CLOSE_PARENTHESIS, newMethodCall, newMethodParameterNames,
		methodParameterNames, "param#");
}

private String _addOrReplaceParameters(
	String lastCharacter, String newMethodCall,
	List<String> newParameterNames, List<String> parameterNames,
	String prefix) {
```

Reply:
@natocesarrego To continue the pattern, I changed the name of the _formatParameters method to _formatMethodParameters.